_includes/header.html

```diff
@@ -108,29 +108,21 @@
 		                    </ul>
 		                </div>
 
-						<div class="module widget-handle language left">
-		                    <ul class="menu">
-		                        <li class="has-dropdown">
-		                            <a href="#">ENG</a>
-		                            <ul>
-		                                <li>
-		                                    <a href="https://translate.google.com/translate?sl=auto&tl=fr&js=y&prev=_t&hl=en&ie=UTF-8&u=https%3A%2F%2Fgci17.fossasia.org%2F&edit-text=">French</a>
-		                                </li>
-		                                <li>
-		                                    <a href="https://translate.google.com/translate?hl=en&sl=auto&tl=es&u=https%3A%2F%2Fgci17.fossasia.org%2F&sandbox=1">Spanish</a>
-		                                </li>
-		                                <li>
-		                                    <a href="https://translate.google.com/translate?hl=en&sl=auto&tl=pt&u=https%3A%2F%2Fgci17.fossasia.org%2F">Portuegese</a>
-		                                </li>
-		                                <li>
-		                                    <a href="https://translate.google.com/translate?hl=en&sl=en&tl=zh-CN&u=https%3A%2F%2Fgci17.fossasia.org%2F">Chinese</a>
-		                                </li>
-		                            </ul>
-		                        </li>
-		                    </ul>
-		                </div>
+						
+						<div class="module-group right">
+							<ul class="menu">
+								<div id="google_translate_element"></div>
+								<script type="text/javascript">
```

Comment:
Move the scripts to footer to reduce the impact of such scripts and make it async as well.